root/layout/components/Footer.js

```diff
@@ -9,87 +9,568 @@
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/How_To"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('How Tos')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/Frequently_Asked_Questions"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('FAQs')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/MusicBrainz_Documentation"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Doc Index')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/Development"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Development')}
+                </a>
+              </li>
+
+            </ul>
+          </div>
+        </div>
+        <div className="row mt-4">
+          <div className="col-md-3 border-top pt-4 d-none d-md-block fs-4">
+            <p>
+              {l('OSS Geek? ')}
+              <a
+                href="https://github.com/metabrainz/musicbrainz-server"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                <span>
+                  {l('Contribute Here')}
+                </span>
+              </a>
+            </p>
+          </div>
+          <div className="col-md-6 border-top pt-4 text-center fs-4">
+            {l('Brought to you by')}
+            <div className="image ms-1 me-1">
+              <img
+                alt="MetaBrainz"
+                height="24"
+                src="../../static/images/meb-icons/MetaBrainz.svg"
+                width="24"
+              />
+            </div>
+            <span>
+              {l('MetaBrainz Foundation')}
+            </span>
```

Comment:
"Brought to you by MetaBrainz Foundation" should be one string; can we maybe inject a `<span />` into the string and display the image with CSS instead?

Reply:
Fixed